core/execution/spot/market.go

```diff
@@ -2891,6 +2894,8 @@ func (m *Market) OnEpochEvent(ctx context.Context, epoch types.Epoch) {
 		feesStats := m.fee.GetFeesStatsOnEpochEnd(quoteAssetQuantum)
 		feesStats.EpochSeq = epoch.Seq
 		feesStats.Market = m.GetID()
+
+		m.banking.RegisterTakerFees(ctx, feesStats.Asset, m.fee.TotalMakerFeesPerParty())
```

Comment:
Minor: its bit confusing for Register _Taker_ Fees function to take input from a call to Total _Maker_ FeesPerParty.
Can we not all them all "TradingFees" because they are they sum of both taker+maker fees?